Lib/glyphsLib/builder/builders.py

```diff
@@ -506,6 +507,25 @@ def _copy_bracket_layers_to_ufo_glyphs(self, bracket_layer_map):
                         )
                         if bracket_comp_name in bracket_glyphs:
                             comp.baseGlyph = bracket_comp_name
+                    # Update kerning groups and pairs, bracket glyphs inherit the
+                    # parent's kerning.
+                    for group, names in ufo_font.groups.items():
+                        if not group.startswith(("public.kern1.", "public.kern2.")):
+                            continue
+                        name_set = set(names)
+                        if glyph_name in name_set and ufo_glyph_name not in name_set:
+                            names.append(ufo_glyph_name)
+                    bracket_kerning = {}
+                    for (first, second), value in ufo_font.kerning.items():
+                        first_match = first == glyph_name
+                        second_match = second == glyph_name
+                        if first_match and second_match:
+                            bracket_kerning[(ufo_glyph_name, ufo_glyph_name)] = value
+                        elif first_match:
+                            bracket_kerning[(ufo_glyph_name, second)] = value
+                        elif second_match:
+                            bracket_kerning[(first, ufo_glyph_name)] = value
+                    ufo_font.kerning.update(bracket_kerning)
```

Comment:
maybe make this whole block a separate method that updates the kerning for a bracket glyph